content/F2015/F15-event-teaparty.md

```diff
@@ -0,0 +1,21 @@
+title: Big CSters Board Game and Tea Party Social
+Date: 2015-09-24
+Category: Events
+Tags: social, big csters
+Slug:  tea-party-F15
+Author: Ziwei Zhao
+Summary:  We'll be meeting up in the CS lounge for a tea party social!
+
+Come out for an evening of tea, snacks and board games with fellow women in CS!
+
+Please bring your own mug if you can!
+
+**About Big CSters:** Our Big CSters program helps connect cis & trans women and non-binary students in computer science and related fields, through events geared towards mentorship and community-building.
```

Comment:
Sorry I did not understand this comment. I copied this line from previous event post.